.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -301,15 +301,15 @@
     reports:
       coverage_report:
         coverage_format: cobertura
-        path: /builds/xsoar/content/artifacts/coverage_report/coverage.xml
+        path: ${CI_PROJECT_DIR}/artifacts/coverage_report/coverage.xml
     expire_in: 30 days
     paths:
-      - /builds/xsoar/content/unit-tests
-      - /builds/xsoar/content/artifacts/*
-      - /builds/xsoar/content/pipeline_jobs_folder/*
+      - ${CI_PROJECT_DIR}/unit-tests
+      - ${CI_PROJECT_DIR}/artifacts/*
+      - ${CI_PROJECT_DIR}/pipeline_jobs_folder/*
     when: always
   services:
-    - name: docker.art.code.pan.run/build-tools--image-dind:20.10.12-dind
+    - name: ${DOCKER_IO}/library/docker:20.10.12-dind
```

Comment:
Didn't we say we'd use the vanilla `dind` image?

Reply:
I thought we agreed to keep the specific version of the docker image. wdyt @kobymeir ?

Reply:
@tkatzir @michal-dagan we decided to keep the version locked on a specific version. what we have now in gitlab.